lib/kino_maplibre/map_cell.ex

```diff
@@ -363,6 +363,7 @@ defmodule KinoMapLibre.MapCell do
 
   defp is_geometry?(%module{}) when module in @geometries, do: true
   defp is_geometry?("http" <> url), do: url |> String.split(".") |> List.last() == "geojson"
+  defp is_geometry?("topojson" <> url), do: url |> String.split(".") |> List.last() == "json"
```

Comment:
I see... I think we should do something like this instead:

```suggestion
  defp is_geometry?(url) when is_binary(url), do: URI.parse(url).scheme in ~w(http https topojson)
```